Update vault guide - Added vault-lab instance reference (#254)
bcgov/platform-developer-docs: src/docs/secrets-management/vault-getting-started-guide.md

# Vault getting started guide

> The Platform Services team has deployed Hashicorp's Vault application with disaster recovery in our Gold service clusters (Gold Kamloops and Gold Calgary), the service is available to all BC Government development teams on **all clusters** (silver, gold/golddr, emerald).

Reviewer: ```suggestion
The Platform Services team has deployed Hashicorp's Vault application with disaster recovery in our Gold service clusters (Gold Kamloops and Gold Calgary), the service is available to all B.C. government development teams on **all clusters** including: Silver, Gold/Golddr and Emerald.
```

The Vault service is available at [https://vault.developer.gov.bc.ca/](https://vault.developer.gov.bc.ca/).

> Lab clusters, such as CLAB, use a separate Vault service available at [https://vault.developer.gov.bc.ca/](https://vault-lab.developer.gov.bc.ca/).

Reviewer 2: ```suggestion
If you are running a test project in the lab clusters, such as CLAB, use a separate Vault service available at [https://vault-lab.developer.gov.bc.ca/](https://vault-lab.developer.gov.bc.ca/). Please note that the lab Vault instance is meant to be used for testing purposes by the Platform Services team, there could be frequent changes and outages with the service!
```

> If you are running a test project in the lab clusters, such as KLAB, use a separate Vault service available at [https://vault-lab.developer.gov.bc.ca/](https://vault-lab.developer.gov.bc.ca/). Please note that the lab Vault instance is meant to be used for testing purposes by the Platform Services team, there could be frequent changes and outages with the service!

Reviewer: ```suggestion
If you are running a test project in the lab clusters, such as KLAB, use a separate Vault service available at [https://vault-lab.developer.gov.bc.ca/](https://vault-lab.developer.gov.bc.ca/).  The lab Vault instance is meant to be used for testing purposes by the Platform Services team, there could be frequent changes and outages with the service!
```
